printer_mods/Gearnut/V0_Misumi_LED_Mount/README.md

```diff
@@ -0,0 +1,21 @@
+V0 Misumi Led mount
+
+![Alt text](https://github.com/GearNut/VoronUsers/blob/master/printer_mods/Gearnut/V0_Misumi_LED_Mount/Images/Cover.jpg)
+
+These were modified from the Misumi Led clips(https://www.teamfdm.com/files/file/338-misumi-led-corners/) as I like the design with them in my V2.4 They should fit most WS2812b Led strips you can find online though If it has the silicone cover on them you may need to peel them off or file down the inside track a touch.
```

Comment:
Is this mod available in this github repo? we'd prefer you link to it from here instead.

Reply:
Unfortunately I dont think it is. I Have looked around and cant find it and the teamfdm link has a git hub link on it but the link is dead and ramalama2 has it in his depreciated folder on his guthub. Team fdm was the most user friendly site I could find to link.

Reply:
Isn't it this one? https://github.com/VoronDesign/VoronUsers/tree/master/printer_mods/eddie/LED_Bar_Clip

Reply:
So the first link you used was the correct one and the file I started with. just used a basic 1515 stl file from somewhere(i think thingiverse) for the 1515 clip and just modelled the two together with some tweaks for fitment. 

so ill update it to the first one as thats still the stl I use in my V2 and VT